examples/data-sources/stackit_argus_instance/data-source.tf

```
@@ -0,0 +1,4 @@
+data "stackit_argus_instance" "example" {
+  name       = "example"
```

the data resource for argus instance requires an `id` not a `name`
```suggestion
  id       = "example"
```